src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -181,6 +186,24 @@ public void assertMatchingGenotypesFromTileDB(File input, File expected, Locatab
         runGenotypeGVCFSAndAssertSomething(genomicsDBUri, expected, NO_EXTRA_ARGS, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes, reference);
     }
 
+    @DataProvider(name="VCFdata")
+    public Object[][] getVCFdata() {
+        return new Object[][]{
+                {new File[]{getTestFile("../CombineGVCFs/YRIoffspring.chr20snippet.g.vcf"), NA12878_HG37}, getTestFile("newMQcalc.genotyped.vcf"), "20:10000000-10100000", NO_EXTRA_ARGS, b37_reference_20_21}
+        };
+    }
+
+    @Test (dataProvider = "VCFdata")
+    public void assertMatchingGenotypesFromGenomicsDB_vidmapHack(File[] inputs, File expected, String interval, List<String> additionalArguments, String reference) throws IOException {
```

Comment:
Could you add a comment explaining this test? It doesn't seem related to the rest of the branch.

Reply:
I goofed and copied the genotypes comparison when I meant do compare annotations.  I fixed the name and made sure MQ is actually being compared.  (FYI the vidmap hack is because I need to change the GDB json since I changed the MQ annotation name.  I'll update this once the protobuf update goes in.)